### ISSUE TITLE
Vehicle images not showing in inventory view

### ISSUE THREAD

**Issue by mjburn0428** [mjburn0428/CSE-340#1]
Can't get vehicle images to show in inventory view.

**Comment by mjburn0428** [comment]
Still haven't figured out the images. 

From my instructor

"[Saturday 4:18 PM] Stephenson, Daniel
The path to your images is wrong. In your database, you have it listed as /images/vehicles/NAME_OF_CAR_IMAGE, but in your code, you are still pointing at /images/NAME_OF_CAR_IMAGE
The name of the car image doesn't exist in your local vehicles directory.
 
[Saturday 4:18 PM] Stephenson, Daniel
For example, you have (in your custom classification) the first image listed as bat-tn.jpg
[Saturday 4:18 PM] Stephenson, Daniel
But the image in your vehicles directory is batmobile-tn.jpg
[Saturday 4:19 PM] Stephenson, Daniel
Basically, the path and name in your database need to match what you have in your images/vehicles directory locally.
[Saturday 4:19 PM] Stephenson, Daniel
Does that make sense?"

**Comment by mjburn0428** [comment]
Tried following the instructions of linking the images. Still not working. :(

**Comment by mjburn0428** [comment]
Deployed to Render. However images aren't loading. Will look at the code for images to navigate to the proper file paths.

**Comment by mjburn0428** [comment]
Fixed. Had to redo the entire database. Found a simpler solution. :)
